devbox.go

```diff
@@ -20,6 +21,26 @@ import (
 // configFilename is name of the JSON file that defines a devbox environment.
 const configFilename = "devbox.json"
 
+var errNoDevboxJSON = func(dir string) error {
```

Comment:
feels a bit nonstandard to declare function in this way?

Reply:
oh yeah, its highly non-standard, not a bit ;)

I'll edit it to be a regular function called `resolveMissingDevboxJsonError`